code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -0,0 +1,236 @@
+//Objects that spawn ghosts in as a certain role when they click on it, i.e. away mission bartenders.
+
+//Preserved terrarium/seed vault: Spawns in seed vault structures in lavaland. Ghosts become plantpeople and are advised to begin growing plants in the room near them.
+/obj/effect/mob_spawn/human/seed_vault
+	name = "preserved terrarium"
+	desc = "An ancient machine that seems to be used for storing plant matter. The glass is obstructed by a mat of vines."
+	mob_name = "a lifebringer"
+	icon = 'icons/obj/lavaland/spawners.dmi'
+	icon_state = "terrarium"
+	density = TRUE
+	roundstart = FALSE
+	death = FALSE
+	mob_species = /datum/species/pod
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are a sentient ecosystem - an example of the mastery over life that your creators possessed. Your masters, benevolent as they were, created uncounted \
+	seed vaults and spread them across the universe to every planet they could chart. You are in one such seed vault. Your goal is to cultivate and spread life wherever it will go while waiting \
+	for contact from your creators. Estimated time of last contact: Deployment, 5x10^3 millennia ago.</b>"
```

Comment:
good good good this means that i can check off the 'lavaland is the remains of clown planet' box from my big theory wall